libnm/nm-device-p2p-wifi.c

```diff
@@ -410,7 +389,11 @@ finalize (GObject *object)
 {
 	NMDeviceP2PWifiPrivate *priv = NM_DEVICE_P2P_WIFI_GET_PRIVATE (object);
 
+	g_clear_object (&priv->proxy);
 	g_free (priv->hw_address);
+	g_byte_array_unref (priv->wfd_ies);
```

Comment:
`priv->wfd_ies` might be NULL here. So this should probably be `g_clear_pointer (&priv->wfd_ies. g_byte_array_unref);` or just an if as with the `peers`.